addons/payment/views/payment_templates.xml

```diff
@@ -244,14 +244,19 @@
                             <!-- === Token name === -->
                             <span class="payment_option_name" t-esc="token.name"/>
                             <!-- === "V" check mark === -->
-                            <i t-if="token.verified" class="fa fa-check text-success"
-                               title="This payment method has been verified by our system."
-                               role="img"
-                               aria-label="Ok"/>
-                            <i t-else="" class="fa fa-check text-muted"
-                               title="This payment method has not been verified by our system."
-                               role="img"
-                               aria-label="Not verified"/>
+                            <t t-if="token.acquirer_id.name=='Test'">
```

Comment:
Don't rely on users not editing this field. Use the `provider` instead: it's the same for all records of the payment acquirer "Test".